api/route/pokemons.route.spec.ts

```diff
@@ -184,3 +185,834 @@ describe("GET /pokemons/random", () => {
+
+  beforeEach(async () => {
+    await prisma.userPokemon.deleteMany({});
+
+    await prisma.pokemon.deleteMany({});
+    await prisma.user.deleteMany({});
+
+    const user = await prisma.user.create({
+      data: {
+        email: "test@example.com",
+      },
+    });
+    userId = user.id;
+  });
+
+  it("should return paginated results with default values", async () => {
+    await prisma.pokemon.createMany({
+      data: [
+        {
+          id: "1",
+          name: "Bulbasaur",
+          hp: 45,
+          attack: 49,
+          defense: 49,
+          speed: 45,
+          description: "A grass Pokémon",
+          image: "image_url",
+          type: ["Grass"],
+          abilities: ["Overgrow"],
+          height: "0.7 m",
+          weight: "6.9 kg",
+        },
+        {
+          id: "2",
+          name: "Charmander",
+          hp: 39,
+          attack: 52,
+          defense: 43,
+          speed: 65,
+          description: "A fire Pokémon",
+          image: "image_url",
+          type: ["Fire"],
+          abilities: ["Blaze"],
+          height: "0.6 m",
+          weight: "8.5 kg",
+        },
+        {
+          id: "3",
+          name: "Squirtle",
+          hp: 44,
+          attack: 48,
+          defense: 65,
+          speed: 43,
+          description: "A water Pokémon",
+          image: "image_url",
+          type: ["Water"],
+          abilities: ["Torrent"],
+          height: "0.5 m",
+          weight: "9.0 kg",
+        },
+        {
+          id: "4",
+          name: "Caterpie",
+          hp: 40,
+          attack: 50,
+          defense: 60,
+          speed: 70,
+          description: "A water Pokémon",
+          image: "image_url",
+          type: ["Water"],
+          abilities: ["Torrent"],
+          height: "0.5 m",
+          weight: "9.0 kg",
+        },
+        {
+          id: "5",
+          name: "Pikachu",
+          hp: 44,
+          attack: 48,
+          defense: 65,
+          speed: 43,
+          description: "A electric Pokémon",
+          image: "image_url",
+          type: ["Electric"],
+          abilities: ["Torrent"],
+          height: "0.5 m",
+          weight: "9.0 kg",
+        },
+      ],
+    });
+
+    const response = await request(app).post("/pokemons").send({}).expect(200);
+
+    expect(response.body[0]).toHaveProperty("id");
+    expect(response.body[0]).toHaveProperty("name");
+    expect(response.body[0]).toHaveProperty("type");
+    expect(response.body[0]).toHaveProperty("hp");
+    expect(response.body[0]).toHaveProperty("attack");
+    expect(response.body[0]).toHaveProperty("defense");
+    expect(response.body[0]).toHaveProperty("speed");
+    expect(response.body[0]).toHaveProperty("description");
+    expect(response.body[0]).toHaveProperty("image");
+    expect(response.body[0]).toHaveProperty("abilities");
+    expect(response.body[0]).toHaveProperty("height");
+    expect(response.body[0]).toHaveProperty("weight");
```

Comment:
you check that the response has the fields but how do you know that the length is the one you expect??
how do you know that  the values are valid?